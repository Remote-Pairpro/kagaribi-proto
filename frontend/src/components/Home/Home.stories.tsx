import { storiesOf } from "@storybook/react";
import * as React from "react";
import { Home } from "./Home";

storiesOf("Home", module).add("default", () => {
    return <Home />;
});
