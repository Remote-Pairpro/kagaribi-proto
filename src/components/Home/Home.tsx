import * as React from "react";

export const Home = () => (
    <div>
        <h1>Home</h1>
        <p>ようこそ Kagaribi へ</p>
    </div>
);

export default Home;
