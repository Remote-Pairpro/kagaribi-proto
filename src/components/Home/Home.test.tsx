import { render } from "enzyme";
import React from "react";
import renderer from "react-test-renderer";
import Home from "./Home";

describe("Home Test", () => {
    it("static contents test", () => {
        const wrapper = render(<Home />);
        expect(wrapper.text()).toContain("Home");
        expect(wrapper.text()).toContain("ようこそ Kagaribi へ");
    });

    it("renders Home correctly", () => {
        const tree = renderer.create(<Home />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
