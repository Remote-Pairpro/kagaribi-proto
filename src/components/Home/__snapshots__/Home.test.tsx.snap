// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Test renders Home correctly 1`] = `
<div>
  <h1>
    Home
  </h1>
  <p>
    ようこそ Kagaribi へ
  </p>
</div>
`;
